src/Venturecraft/Revisionable/RevisionableTrait.php

```diff
@@ -244,21 +426,16 @@ protected function dbInsert(array $revisions)
     }
 
     /**
-     * Attempt to find the user id of the currently logged in user
-     * Supports Cartalyst Sentry/Sentinel based authentication, as well as stock Auth
-     **/
+     * Attempt to find the user id of the currently logged in user.
+     *
+     * @return string|null
+     */
     private function getUserId()
     {
-        try {
-            if (class_exists($class = '\Cartalyst\Sentry\Facades\Laravel\Sentry')
-                || class_exists($class = '\Cartalyst\Sentinel\Laravel\Facades\Sentinel')
-            ) {
-                return ($class::check()) ? $class::getUser()->id : null;
-            } elseif (\Auth::check()) {
-                return \Auth::user()->getAuthIdentifier();
-            }
-        } catch (\Exception $e) {
-            return null;
+        $jwtAuth = App::make('Tymon\JWTAuth\JWTAuth');
```

Comment:
@xiphiaz `\Auth` is a facade for `Illuminate\Auth\Guard`. `user()` in guard doesn't use a resolver, but is hardcoded to get it from the session. The update we made only made the user available to the request object. If we want `Guard` to be able to get the user as well, we can do that by extending the `Guard` class.

But you're right we can get the user much simpler now. I should be able to take away the dependency on JWTAuth and get the user with `\Request::user()` that will work what we currently have.

Depending on how the new ACL works under the hood, using Guard or Request for user resolving, it might be required to extend the Guard class though anyway. 

I'll change it to use `Request` to begin with.